openassessment/templates/openassessmentblock/peer/oa_peer_cancelled.html

```diff
@@ -8,7 +8,7 @@
 {% endblock %}
 
 {% block button %}
-    <button class="ui-slidable" aria-expanded="false" disabled="disabled">
+    <button class="ui-slidable" aria-expanded="false" disabled>
```

Comment:
The most correct syntax is simply _disabled_, though other variations seem to work in most browsers. Went ahead and cleaned this up while searching for usages of disabled.